GoogleForCreators/web-stories-wp
# Workspace: Update Pill Group and Collapse Button in Text Sets Panel, media 3p Panel, and Page Layouts Panel

## Context

Updating the editor to use components from the design system.

[Designs:](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=257%3A817)
![image](https://user-images.githubusercontent.com/22185279/113188671-92385300-9217-11eb-94dc-469a54d8b129.png)

## Summary

Updates the Text Sets Pane and the Page Layouts Pane
- Create new `ChipGroup` based on the old `PillGroup` in the editor

## Relevant Technical Choices

~We need to attach a ref to each pill, so it was necessary to set up its own pill group rather than use the one from the design system. Another option would have been to add a prop that would pass refs to each pill in the `PillGroup` from the design system.~ These are supposed to be `Chips` not `Pills`!

Added a new `ChipGroup` since the `PillGroup` is used in many places in the editor still.

## To-do

none

## User-facing changes

|Before|After|
|--|--|
|![before-pills](https://user-images.githubusercontent.com/22185279/113189173-230f2e80-9218-11eb-8762-6319d8e15204.gif)|![pills](https://user-images.githubusercontent.com/22185279/113189391-649fd980-9218-11eb-86b7-79d588b3d5e2.gif)|
|![page-layouts-before](https://user-images.githubusercontent.com/22185279/113321236-a0967580-92d0-11eb-9137-9a8d449b8f80.gif)|![page-layouts-chips](https://user-images.githubusercontent.com/22185279/113321249-a2f8cf80-92d0-11eb-88db-2dc15a6fc591.gif)|
|![before](https://user-images.githubusercontent.com/22185279/113606617-7fea5a80-9605-11eb-90dc-b1afca453087.gif)|![after](https://user-images.githubusercontent.com/22185279/113606677-8e387680-9605-11eb-9b36-64c8424fc2f4.gif)|


## Testing Instructions

1. Go to editor.
2. Open Text panes in the library.
3. Wait for text panes to load. Should see the pills. Interact with the pills.
4. Should be keyboard accessible.
5. Try the same thing in the Page layouts

### QA






This PR can be tested by following these steps:

1. Go to editor.
2. Open Text panes in the library.
3. Wait for text panes to load. Should see the pills. Interact with the pills.
4. Should be keyboard accessible.
5. Try the same thing in the Page layouts

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6873 
Fixes #6874 
Fixes #7052
